### ISSUE TITLE
Article 'Date' not formatting correctly

### ISSUE THREAD

**Issue by zeim839** [ufosc/Club_Website_2#150]
**Preview:**
<img width="843" alt="image" src="https://user-images.githubusercontent.com/50573884/236656372-6d09735d-f059-4971-b9ed-50169c92767f.png">

**Help Wanted:**
The "Date" string of any article page (navigate to the /blog page) should display a date in the `MONTH DAY, YYYY` format. A similar solution has already been implemented [here](https://github.com/ufosc/Club_Website_2/blob/main/views/components/blog-grid.ejs#L18). I imagine that replacing [views/article.ejs:22](https://github.com/ufosc/Club_Website_2/blob/main/views/article.ejs#LL22C21-L22C47) to the same thing should resolve this issue:
```ejs
<h3>Date: <%= date %></h3>
```
should become:
```ejs
<h3>Date: <%= date.toLocaleDateString('en-us', { month: "long", day: "numeric",
                year: "numeric" }) %></h3>
```

**Comment by pranit-yawalkar** [comment]
@zeim839 Please assign me

**Comment by zeim839** [comment]
Sure, @pranit-yawalkar

**Comment by zeim839** [comment]
Implemented in #151
